dart/lib/src/locator.dart

```diff
@@ -0,0 +1,89 @@
+import 'dart:ffi';
+import 'dart:io';
+
+/// Attempts to locate the MinifyHtml dynamic library.
+///
+/// Throws [MinifyHtmlLocatorError] if the dynamic library could not be found.
+DynamicLibrary loadDynamicLibrary() {
```

Comment:
I'm looking at the examples on the Dart lang docs and they don't seem to try to find the package root. Is this script not always in a fixed location relative to the package?

Reply:
I've updated the code to look for the library in the script directory and current directory and their parents. It's just a pattern I've noticed that other programs tend to do when looking for dynamic libraries on desktops.